### PR TITLE
Fix: Fixed crash when manipulating invalid images

### LINKED ISSUE #14025: Bug: COMException in BitmapHelper.RotateAsync
### Description

BitmapHelper.RotateAsync (String filePath, BitmapRotation rotation)
System.Runtime.InteropServices.COMException

### Steps To Reproduce

1. Rotate image
2. App crashes

### Requirements

- Fix crash when rotating images

### Files Version

v3.0.10

### Windows Version

Windows 11

### Log File

```
Files.App.Helpers
BitmapHelper.RotateAsync (String filePath, BitmapRotation rotation)
Files.App.Actions
BaseRotateAction.ExecuteAsync ()
Files.App.Data.Commands.CommandManager
ActionCommand.Execute (Object parameter)
```

### PR DESCRIPTION
**Pull request by hishitetsu**
**Resolved / Related Issues**
- [X] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #14025

**Validation**
How did you test these changes?
- [X] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [X] Are there any other steps that were used to validate these changes?
   1. Create a 0 byte file that has a .bmp extension.
   2. Try to rotate the file.
   3. Now an error dialog should be displayed.

### CODE REVIEW

File: src/Files.App/Helpers/Application/AppLifecycleHelper.cs [hunk 1]
```diff
@@ -317,7 +317,7 @@ public static void HandleAppUnhandledException(Exception? ex, bool showToastNoti
 				{
 					await Launcher.LaunchUriAsync(new Uri("files-uwp:"));
 				})
-				.Wait(1000);
+				.Wait(100);
```
Comment (hishitetsu) [inline src/Files.App/Helpers/Application/AppLifecycleHelper.cs:320]: This is not directly related to fixing the crash, but it does make it easier for an automatic restart to succeed in the event of a crash.

Comment (yaira2) [reply src/Files.App/Helpers/Application/AppLifecycleHelper.cs:320]: Thanks! I noticed that it hasn't been restarting that often.

**Review by yaira2 — APPROVED** [review]
LGTM
